sunkit_image/tests/conftest.py

```diff
@@ -173,3 +173,332 @@ def aia_171(request):
+                43798.25,
+                41219.75,
+                42207.5,
+                41275.25,
+                39542.0,
+                34676.75,
+                475.25,
+                -1000.0,
+            ],
+            [
+                -1000.0,
+                -1000.0,
+                28699.25,
+                36770.75,
+                38501.0,
+                43415.75,
+                41420.75,
+                43828.25,
+                42236.0,
+                43340.0,
+                40098.5,
+                38181.5,
+                36518.0,
+                27592.25,
+                -1000.0,
+                -1000.0,
+            ],
+            [
+                -1000.0,
+                -1000.0,
+                -1000.0,
+                28475.0,
+                33800.75,
+                36760.25,
+                40776.5,
+                39202.25,
+                39293.75,
+                38585.0,
+                36566.0,
+                34690.25,
+                27785.75,
+                -1000.0,
+                -1000.0,
+                -1000.0,
+            ],
+            [
+                -1000.0,
+                -1000.0,
+                -1000.0,
+                -1000.0,
+                19496.75,
+                30044.0,
+                33616.25,
+                34552.25,
+                33396.5,
+                33146.75,
+                30135.5,
+                983.75,
+                -1000.0,
+                -1000.0,
+                -1000.0,
+                -1000.0,
+            ],
+            [
+                -1000.0,
+                -1000.0,
+                -1000.0,
+                -1000.0,
+                -1000.0,
+                -1000.0,
+                261.5,
+                393.5,
+                383.75,
+                262.25,
+                -1000.0,
+                -1000.0,
+                -1000.0,
+                -1000.0,
+                -1000.0,
+                -1000.0,
+            ],
+        ]
+    )
+    # Create the header with appropriate metadata
+    header = {
+        "simple": "True",
+        "bitpix": -64,
+        "naxis": 2,
+        "naxis2": 16,
+        "naxis1": 16,
+        "date": "2024-05-12T09:54:44",
+        "date-obs": "2024-05-08T16:54:35.10",
+        "wavelnth": 6173.0,
+        "bunit": "DN/s",
+        "ctype1": "HPLN-TAN",
+        "ctype2": "HPLT-TAN",
+        "crpix1": 8.45965528515625,
+        "crpix2": 8.5173759453125,
+        "crval1": 0,
+        "crval2": 0,
+        "cdelt1": 129.034752,
+        "cdelt2": 129.034752,
+        "cunit1": "arcsec",
+        "cunit2": "arcsec",
+        "crota2": 179.929596,
+        "rsun_obs": 950.800171,
+        "rsun_ref": 696000000,
+        "wcsname": "Helioprojective-cartesian",
+        "dsun_obs": 150989471585.0,
+        "crln_obs": 330.716064,
+        "crlt_obs": -3.348247,
+        "car_rot": 2284,
+    }
+
+    return sunpy.map.GenericMap(data, header)
```

Comment:
Can this file not be saved as a file via Maps save function?

Reply:
it can be saved